### DISCUSSION TITLE
Is actually possible to use standalone users operator with AWS MSK?

### DISCUSSION THREAD

**Discussion by leunamnauj** [strimzi/strimzi-kafka-operator#9051] [Q&A]
I'm testing out standalone operators with MSK (scram-sha-512 auth). 
Topics op is working fine but sadly is not the case with user op. I've found some posts on this discussions saying that *STRIMZI_KAFKA_ADMIN_CLIENT_CONFIGURATION* env var have to be used for SASL but I couldnt found any example about how to do it.
On the other hand, I also found [this](https://github.com/orgs/strimzi/discussions/6998#discussioncomment-3383147) that pretty much states that is not possible to be used.

Has anyone succeeded on this?
Thanks in advance

**Comment by scholzj** [comment]
> On the other hand, I also found [this](https://github.com/orgs/strimzi/discussions/6998#discussioncomment-3383147) that pretty much states that is not possible to be used.

That is quite old (The User Operator was completely rewritten in Strimzi 0.33). And I do not think it says that it was not possible at that time. It just says that there is no official effort or testing done to specifically support it.

**Comment by leunamnauj** [comment]
Thanks @scholzj for your quick response.
Therefore, is not possible (for the moment) to use scram users on standalone users operator?

**Reply by scholzj** [reply]
I don't know, sorry. I personally based on my knowledge of the code think it should be possible using the `STRIMZI_KAFKA_ADMIN_CLIENT_CONFIGURATION` variable. But we never tried it, it was never designed specifically for that use-case, and there is no testing for such use-case being part of our code base. That does not mean it is not possible. I'm not entirely sure if other users figured it out or not. I was really just trying to reiterate the previous comment you linked to.

**Reply by leunamnauj** [reply]
Thanks a lot
